website/static/js/citation-settings.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
```

Comment:
JS modules should be named in `lowerCamelCase`. @samchrisinger Can you change this post-merge?